src/main/java/kr/co/antoon/graph/infrastructure/GraphScoreSnapshotRepository.java

```diff
@@ -19,4 +19,6 @@ public interface GraphScoreSnapshotRepository extends JpaRepository<GraphScoreSn
     Optional<GraphScoreSnapshot> findTop1ByWebtoonIdOrderBySnapshotTimeDesc(Long webtoondId);
 
     List<GraphScoreSnapshot> findAllByWebtoonIdAndSnapshotTimeBetweenOrderByCreatedAtAsc(Long webtoonId, LocalDateTime start, LocalDateTime end);
+
+    GraphScoreSnapshot findTop1ByWebtoonIdAndSnapshotTimeBetweenOrderByCreatedAtDesc(Long webtoonId, LocalDateTime start, LocalDateTime end);
```

Comment:
```suggestion
    GraphScoreSnapshot findTopOneByWebtoonIdAndSnapshotTimeBetweenOrderByCreatedAtDesc(Long webtoonId, LocalDateTime start, LocalDateTime end);
```
이 부분도 숫자가 아닌 영어로 표기 하는 게 더 좋을 것 같습니다.

Reply:
넵 감사합니다 👍